extremeflash/ws_ap3935i.py

```diff
@@ -0,0 +1,238 @@
+        )
+
+    # https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=3aef61060e3f51aa43fe494d5ff173e81dd43003
+    boot_openwrt_params = b"setenv boot_openwrt 'bootm 0x42000000'"
+
+    if "boot_openwrt" in printenv_return:
+        logging.debug("Found existing U-Boot boot_openwrt parameter. Verifying.")
+        existing_boot_openwrt_params = re.search(r"boot_openwrt=(.*)\r\n", printenv_return)
+        if not existing_boot_openwrt_params:
+            raise RuntimeError("Unable to parse detected boot_openwrt paramter")
+
+        if boot_openwrt_params.decode("ascii") != existing_boot_openwrt_params.group(1):
+            # Some AP3825i had wrong and/or outdated boot_openwrt parameters in the past.
+            logging.warning(f"Overwriting unexpected param for 'boot_openwrt': {existing_boot_openwrt_params}")
+        else:
+            # do not set anything if we found boot_openwrt
+            # TODO: should we check if bootcmd is also set correctly?
+            logging.debug("Existing U-Boot boot_openwrt parameter looks good.")
+            return
+    else:
+        logging.info("Did not find boot_openwrt in U-Boot parameters. Setting it.")
+
+    write_to_serial(ser, b'setenv boot_openwrt "' + boot_openwrt_params + b'"\n')
+    time.sleep(0.5)
+
+    write_to_serial(ser, b'setenv bootcmd "run boot_openwrt"\n')
+    time.sleep(0.5)
+
+    if ws_ap3710i.DRYRUN:
+        logging.info("dryrun: Skipping saveenv")
+        return
+
+    ser.write(b"saveenv\n")
+    time.sleep(2)
+    saveenv_return = ser.read(ser.in_waiting).decode("ascii")
+    debug_serial(saveenv_return)
+
+    if "Writing to Flash" not in saveenv_return:
+        raise RuntimeError("saveenv did not successfully write to flash")
+
+
+def boot_via_tftp(
+    ser: serial.Serial,
+    tftp_ip: ipaddress.IPv4Interface | ipaddress.IPv6Interface,
+    tftp_file: str,
+    new_ap_ip: ipaddress.IPv4Interface | ipaddress.IPv6Interface,
+):
+    new_ap_ip_str = str(new_ap_ip.ip).encode("ascii")
+    new_ap_netmask_str = str(new_ap_ip.netmask).encode("ascii")
+    tftp_ip_str = str(tftp_ip.ip).encode("ascii")
+
+    write_to_serial(ser, b"setenv ipaddr " + new_ap_ip_str + b"\n")
+    write_to_serial(ser, b"setenv netmask " + new_ap_netmask_str + b"\n")
+    write_to_serial(ser, b"setenv serverip " + tftp_ip_str + b"\n")
+    write_to_serial(ser, b"setenv gatewayip " + tftp_ip_str + b"\n")
+    write_to_serial(ser, b"tftpboot 0x42000000 " + tftp_ip_str + b":" + tftp_file.encode("ascii") + b"\n")
+    # wait until TFTP transfer is complete
+    while ws_ap3710i.event_keep_serial_active.is_set():
+        line = readline_from_serial(ser)
+
+        if "Bytes transferred" in line:
+            time.sleep(1)
+            break
+    # Note: We must step through the `bootm` process manually to avoid fdt relocation.
+    # https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=3aef61060e3f51aa43fe494d5ff173e81dd43003
+    #write_to_serial(ser, b"interrupts off\n")
```

Comment:
either remove the interrupts line or change it to be a human readable comment (as to why you cannot use interrupts for example).

From pylint:
```
./extremeflash/ws_ap3935i.py:127:5: E265 block comment should start with '# '
    #write_to_serial(ser, b"interrupts off\n")
```